### PR TITLE
b+c components - api docs

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
b+c components - api docs

## Changelog
b+c components - api docs

### CODE REVIEW

File: src/components/carousel/carousel.api.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     {
       "name": "containerMarginHorizontal",
       "type": "number",
-      "description": "Horizontal margin for the carousel container"
+      "description": "Horizontal margin for the carousel container",
+      "default": "true"
```
Comment (lidord-wix) [inline src/components/carousel/carousel.api.json:26]: The type is `number` and the default value is `true`?

File: src/components/colorPicker/colorPicker.api.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "name": "colors",
+      "type": "string[]",
+      "description": "Array of colors for the picker's color palette (hex values)"
+    },
+    {"name": "value", "type": "string", "description": "The value of the selected swatch"},
+    {
+      "name": "animatedIndex",
+      "type": "number",
+      "description": "The index of the item to animate at first render",
+      "note": "Default is last"
```
Comment (lidord-wix) [inline src/components/colorPicker/colorPicker.api.json:21]: looks like `default` is a better key here

File: src/components/colorPicker/colorPicker.api.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    {
+      "name": "onValueChange",
+      "type": "(value: string, options: object) => void",
+      "description": "Callback for the picker's color palette change"
+    },
+    {
+      "name": "accessibilityLabels",
+      "type": "{\n  addButton?: string,\n  dismissButton?: string,\n  doneButton?: string,\n  input?: string}",
+      "description": "Accessibility labels as an object of strings",
+      "default": "{\n  addButton: 'add custom color using hex code',\n  dismissButton: 'dismiss',\n  doneButton: 'done',\n  input: 'custom hex color code'\n}"
```
Comment (lidord-wix) [inline src/components/colorPicker/colorPicker.api.json:32]: Will it break the line in `.md` file?

File: src/components/connectionStatusBar/connectionStatusBar.api.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "connectionStatusBar",
```
Comment (lidord-wix) [inline src/components/connectionStatusBar/connectionStatusBar.api.json:2]:
```suggestion
  "name": "ConnectionStatusBar",
```

File: src/components/connectionStatusBar/connectionStatusBar.api.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  "category": "basic",
+  "description": "Top bar to show a 'no internet' connection status",
+  "notes": [
```
Comment (lidord-wix) [inline src/components/connectionStatusBar/connectionStatusBar.api.json:5]: I think we decided to call it `note`

File: src/components/colorPicker/ColorPalette.api.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+      "type": "boolean",
+      "description": "Whether the colors pagination scrolls in a loop",
+      "default": "true"
+    },
+    {"name": "numberOfRows", "type": "number", "description": "The number of color rows from 2 to 5", "default": "3"},
+    {
+      "name": "animatedIndex",
+      "type": "number",
+      "description": "The index of the item to animate at first render",
+      "note": "Default is last"
```
Comment (lidord-wix) [inline src/components/colorPicker/ColorPalette.api.json:30]: looks like `default` is a better key here

**Review by lidord-wix — COMMENTED** [review]
Few comments..

File: src/components/carousel/carousel.api.json (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/carousel/carousel.api.json:26]: haha. It' of the 'horizontal' prop. Fixed

File: src/components/colorPicker/colorPicker.api.json (hunk 2 above)
Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:21]: No. "last" is not a value, it means the last item in the array.

File: src/components/colorPicker/colorPicker.api.json (hunk 3 above)
Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:32]: I hope

File: src/components/connectionStatusBar/connectionStatusBar.api.json (hunk 5 above)
Comment (Inbal-Tish) [reply src/components/connectionStatusBar/connectionStatusBar.api.json:5]: Yes. Ethan just told me. Fixed

File: src/components/colorPicker/ColorPalette.api.json (hunk 6 above)
Comment (Inbal-Tish) [reply src/components/colorPicker/ColorPalette.api.json:30]: No. "last" is not a value, it means the last item in the array.

File: src/components/colorPicker/colorPicker.api.json (hunk 3 above)
Comment (lidord-wix) [reply src/components/colorPicker/colorPicker.api.json:32]: I think the `<br />` will do the work here

Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:32]: What's the difference and why use markup in a string?

Comment (lidord-wix) [reply src/components/colorPicker/colorPicker.api.json:32]: I tried to play with a `.md` file and I saw that `\n` doesn't break the line..

Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:32]: ok... We are using \n very often, did you run the script and see it doesn't break the lines on the doc site?

Comment (lidord-wix) [reply src/components/colorPicker/colorPicker.api.json:32]: Nope, Ethan said it makes `.md` files from the api files so I guess it'll work the same..
but you can check if it works that way and keep it.

Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:32]: Yes, I saw it in the script. I'll check

Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:32]: \n is not working but not visible, while <br/> is not working and it is visible in the string

Comment (Inbal-Tish) [reply src/components/colorPicker/colorPicker.api.json:32]: I'll leave the \n till we find a solution so we'll know where we need a fix

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**